llvm/test/MC/AMDGPU/gfx11_asm_vop1_t16_err-fake16.s

```diff
@@ -1,89 +0,0 @@
-// RUN: not llvm-mc -triple=amdgcn -mcpu=gfx1100 -mattr=-real-true16,+wavefrontsize32 -show-encoding %s 2>&1 | FileCheck --check-prefix=GFX11 --implicit-check-not=error: %s
```

Comment:
Why do we rename this to gfx11_asm_vop1_fake16_err.s? The convention seems to be adding the `-fake16` suffix to whatever is the name of the non-fake16 version of the test?